packages/dds/tree/src/forest/cursor.ts

```diff
@@ -4,89 +4,203 @@
  */
 
 import { assert } from "@fluidframework/common-utils";
-import { FieldKey, TreeType, Value } from "../tree";
+import { FieldKey, TreeType, UpPath, Value } from "../tree";
 
-export const enum TreeNavigationResult {
-    /** Attempt to navigate cursor to a key or index that is outside the client's view. */
-    NotFound = -1,
+/**
+ * A stateful low-level interface for reading tree data.
+ */
+export interface ITreeCursor {
+    /**
+     * What kind of place the cursor is at.
+     * Determines which operations are allowed.
+     */
 
-    /** Attempt to navigate cursor to a portion of the tree that has not yet been loaded. */
-    Pending = 0,
+    readonly mode: CursorLocationType;
+    /*
+     * True iff the current field or node (depending on mode) is "pending",
+     * meaning that is has not been downloaded.
+     */
+    readonly pending: boolean;
```

Comment:
```suggestion
     */

    readonly pending: boolean;
```

Reply:
Not sure why adding a line her would be better.

Reply:
Consistency: this is the one place in the whole file where you don't have an empty line between members.

Reply:
I made a fix for this. The suggestion added a line after the comment instead of the property.